src/main/java/mafia/mafiatogether/service/dto/ChatResponse.java

```diff
@@ -7,7 +7,7 @@ public record ChatResponse(
         String name,
         String contents,
         Timestamp timestamp,
-        boolean owner
+        boolean isOwner
```

Comment:
```suggestion
        Boolean isOwner
```
이것도 내가 실수한건데 Boolean으로 바꾸는건 어때?